packages/app/src/components/Bookmarks/BookmarkFolderItem.tsx

```diff
@@ -173,9 +173,8 @@ const BookmarkFolderItem: FC<BookmarkFolderItemProps> = (props: BookmarkFolderIt
       }
       return item.root !== targetRoot || item.level >= targetLevel;
     }
-    const bookmarks = bookmarkFolder.bookmarks;
-    const isBookmarkExists = bookmarks.filter(bookmark => bookmark.page._id === item._id).length > 0;
-    if (isBookmarkExists) {
+    const isSameTarget = item.parentFolder && item.parentFolder._id === bookmarkFolder._id;
+    if (isSameTarget) {
```

Comment:
Please rewrite like that.
`if (item.parentFolder != null && item.parentFolder._id === bookmarkFolder._id) {`